tests/network/vmi_networking.go

```diff
@@ -818,9 +811,7 @@ var _ = SIGDescribe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com]
 			}
 
 			BeforeEach(func() {
-				if !tests.HasLiveMigration() {
```

Comment:
Perhaps, we just need to move all migration related tests to the migration_test.go

Reply:
probably a `migration` package with various focused files related to storage, network various vm-configurations?

Reply:
we could just consolidate all of these tests in tests/migration_test.go